ts/nni_manager/main.ts

```diff
@@ -174,25 +174,8 @@ mkDirP(getLogDir())
         console.error(`Failed to create log dir: ${err.stack}`);
     });
 
-async function cleanUp(): Promise<void> {
-    const log: Logger = getLogger();
-    let hasError: boolean = false;
-    try {
-        const nniManager: Manager = component.get(Manager);
-        await nniManager.stopExperiment();
-        const experimentManager: ExperimentManager = component.get(ExperimentManager);
-        await experimentManager.stop();
-        const ds: DataStore = component.get(DataStore);
-        await ds.close();
-        const restServer: NNIRestServer = component.get(NNIRestServer);
-        await restServer.stop();
```

Comment:
it is a little strange that these components are created in this file, but stopped in nnimanager.ts

Reply:
Maybe. But I don't want to make huge refactor at this point.